### DISCUSSION TITLE
Monitoring several netwroks on the same interface

### DISCUSSION THREAD

**Discussion by azxqw** [Security-Onion-Solutions/securityonion#2515] [Unsupported Versions] [answered]
Hi,
I recently installed SO 2, and now that i have it configured i have a question about monitoring different networks.
how do i add networks for SO to monitor? all data in our network is mirrored to the SO server, I just dont know how to configure it in SO

**Comment by dougburks** [comment, marked as answer]
If the traffic is reaching the Security Onion sniffing interface, then Security Onion should already be doing basic metadata logging of that traffic.

If you're asking about the HOME_NET setting, you should have been prompted during Setup and the default should have been RFC1918 private address space. If you need to change that to include some non-private address ranges, please see:
https://docs.securityonion.net/en/2.3/homenet.html
